fe/src/pages/LogIn.vue

```diff
@@ -0,0 +1,81 @@
+                    <v-col cols="12" sm="8" md="4" lg="4">
+                        <v-card elevation="0">
+                            <v-card-title class="text-h3 text-md-h20 text-lg-h4 text-center mb-3">Login</v-card-title>
+                                <v-card-text>
+                                    <v-form>                                    
+                                        <v-text-field 
+                                            v-model="email.value.value"
+                                            label="Email" 
+                                            type="email" 
+                                            :error-messages="email.errorMessage.value" 
+                                            prepend-inner-icon ="mdi-email-outline"
+                                            variant="outlined"
+                                        ></v-text-field>    
+
+                                        <br>
+
+                                        <v-text-field
+                                            v-model="password.value.value" 
+                                            label="Password"
+                                            :error-messages="password.errorMessage.value"
+                                            prepend-inner-icon="mdi-lock-outline"
+                                            variant="outlined"
+                                            :type="visiblePw ? 'text' : 'password'"
+                                            :append-inner-icon="visiblePw ? 'mdi-eye' : 'mdi-eye-off'"
+                                            @click:append-inner="visiblePw = !visiblePw"
+                                        ></v-text-field>
+                                    
+                                        <br>
+
+                                        <v-btn 
+                                            @click="handleLogIn" 
+                                            class="mb-2" 
+                                            color="#B2BEB5" 
+                                            x-large block dark
+                                        >
+                                            Log In
+                                        </v-btn>
+                                    </v-form>
+                                </v-card-text>
+                            </v-card>
+                        </v-col>
+                    </v-row>
+                </v-container>
+        </v-main>
+    </v-app>
+</template>
+
+<script setup>
+
+    import { useField, useForm } from 'vee-validate'
+    import {ref} from 'vue'
+   
+    const { handleLogIn} = useForm({
+      validationSchema: {
+        
+        email (value) {
+          if (/^[a-z.-]+@[a-z.-]+\.[a-z]+$/i.test(value)) return true
+  
+          return 'Invalid email.'
+        },
+        
+        password (value) {
+          if (value?.length >= 8 && /^(?=.*[A-Za-z])(?=.*\d).*$/.test(value)) return true

```

Comment:
Please remove the validation of password and email

we will validate it once the user clicks login and then check if credentials are true if not we display "invalid email or password"